decidim-design/app/views/layouts/decidim/design/_layout.html.erb

```diff
@@ -0,0 +1,92 @@
+<%
+  sections = [{
+    url: decidim_design.root_path,
+    path: false,
+    icon: "home-7-line",
+    text: "Home"
+  },{
+    url: false,
+    path: "foundations",
+    icon: "ruler-line",
+    text: "Foundations"
+  },{
+    url: false,
+    path: "components",
+    icon: "focus-line",
+    text: "Components",
+    extra_items: [
+      {
+        name: "cards",
+        path: components_cards_path
+      }
+    ]
+  }]
+%>
+
+<div class="grid md:grid-cols-12 container min-h-screen before:content-[''] before:absolute before:-z-10 before:w-1/2 before:h-full before:inset-0 before:bg-background">
+  <aside class="col-span-2 py-12 pr-16">
+    <ul class="sticky top-12 space-y-4">
+      <% sections.each do |section| %>
+        <li>
+          <% block = capture do %>
+            <%= icon section[:icon], class: "text-gray fill-current" %>
+            <span class="font-semibold"><%= section[:text] %></span>
+          <% end %>
+
+          <% if section[:url].present? %>
+            <%= link_to section[:url], class: "flex gap-1 items-center hover:underline" do %>
+              <%= block %>
+            <% end %>
+          <% else %>
+            <%= content_tag :span, class: "flex gap-1 items-center" do %>
+              <%= block %>
+            <% end %>
+          <% end %>
+
+          <% items = path_items(section[:path]).concat(section[:extra_items].to_a) %>
+          <% if items.present? %>
+            <ul class="pl-5 pt-4 space-y-4">
+              <% items.each do |item| %>
+                <li class="text-sm text-black cursor-pointer flex items-center gap-1">
+                  <%= link_to item[:name], item[:path], class: "hover:underline first-letter:capitalize" %>
```

Comment:
Maybe we can add some kind of indicator ( ex  make the text bold) so we make sure the user knows the page is in?

Reply:
Done